02-temperature_converter/Temperature.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class temperature {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.print("to which unit (celcus/fahrenheit): ");
+        String unit = input.nextLine();
+        if (unit.equals("celcus")) {
```

Comment:
a good opportunity for equalsIgnoreCase(String anotherString); yours is great, just suggesting this method incase of "Celsius" being inputted.